instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/HistoryRecordingStateChangeObserverTest.kt

```diff
@@ -402,6 +397,7 @@ class HistoryRecordingStateChangeObserverTest :
                     ),
                     mockRoute.routeWaypoints.last()
                 ),
+                relaxedExpectedCoordinates = true
```

Comment:
It's not exactly related to location updates but resolving that issue disclosed another one in this test so I fixed it as well.